.github/workflows/deploy.yml

```diff
@@ -0,0 +1,25 @@
+name: Deploy gh-pages
+
+on:
+  push:
+    branches:
+      - main
+permissions:
+  contents: write
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version: 20
+    - run: >
+        npm run build
```

Comment:
Actually I think you might have to do an `npm install` here first.

Reply:
`npm ci --no-audit` ideally.